Jenkinsfile

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env groovy
+
+REPOSITORY = 'smart-answers'
+
+node {
+  def govuk = load '/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy'
+
+  try {
+    stage('Checkout') {
+      checkout scm
+      govuk.cleanupGit()
+      govuk.mergeMasterBranch()
+      govuk.contentSchemaDependency()
```

Comment:
If smart-answers depends on the govuk-content-schemas, this might be a good time to add the functionality to run the smart-answers tests against changes to the govuk-content-schemas, there are some docs here:
https://github.gds/pages/gds/opsmanual/infrastructure/testing/application-testing.html?highlight=schemas#content-schema-tests